nautobot/extras/templates/extras/job_bulk_edit.html

```diff
@@ -0,0 +1,75 @@
+{% extends 'base.html' %}
+{% load helpers %}
+{% load form_helpers %}
+{% load render_table from django_tables2 %}
+
+{% block content %}
+<h1>{% block title %}Editing {{ table.rows|length }} {{ obj_type_plural|bettertitle }}{% endblock %}</h1>
+{% if form.errors %}
+    <div class="panel panel-danger">
+        <div class="panel-heading"><strong>Errors</strong></div>
+        <div class="panel-body">
+            {{ form.non_field_errors }}
+            {% for field in form %}
+                {% if field.errors %}
+                    <strong class="panel-title">{{ field.label }}</strong>: {{ field.errors }}
+                {% endif %}
+            {% endfor %}
+        </div>
+    </div>
+{% endif %}
+<form action="" method="post" class="form form-horizontal">
+    {% csrf_token %}
+    {% if request.POST.return_url %}
+        <input type="hidden" name="return_url" value="{{ request.POST.return_url }}" />
+    {% endif %}
+    {% for field in form.hidden_fields %}
+        {{ field }}
+    {% endfor %}
+    <div class="row">
+        <div class="col-md-8">
+            <div class="panel panel-default">
+                {% render_table table 'inc/table.html' %}
+            </div>
+        </div>
+        <div class="col-md-4">
+            <div class="panel panel-default">
+                <div class="panel-heading"><strong>Attributes</strong></div>
+                <div class="panel-body">
+                    {% render_field form.enabled %}
+                    {% include "extras/inc/bulk_edit_overridable_field.html" with field=form.grouping override_field=form.clear_grouping_override %}
+                    {% include "extras/inc/bulk_edit_overridable_field.html" with field=form.description override_field=form.clear_description_override %}
+                    {% include "extras/inc/bulk_edit_overridable_field.html" with field=form.approval_required override_field=form.clear_approval_required_override %}
+                    {% include "extras/inc/bulk_edit_overridable_field.html" with field=form.dryrun_default override_field=form.clear_dryrun_default_override %}
+                    {% include "extras/inc/bulk_edit_overridable_field.html" with field=form.hidden override_field=form.clear_hidden_override %}
+                    {% include "extras/inc/bulk_edit_overridable_field.html" with field=form.soft_time_limit override_field=form.clear_soft_time_limit_override %}
+                    {% include "extras/inc/bulk_edit_overridable_field.html" with field=form.time_limit override_field=form.clear_time_limit_override %}
+                    {% include "extras/inc/bulk_edit_overridable_field.html" with field=form.has_sensitive_variables override_field=form.clear_has_sensitive_variables_override %}
+                    {% include "extras/inc/bulk_edit_overridable_field.html" with field=form.task_queues override_field=form.clear_task_queues_override %}
+                </div>
+            </div>
+		    <div class="form-group text-right">
```

Comment:
Following previous comment, this is the only part of `generic/object_bulk_update.html` that you'd need to override, right?